### ISSUE TITLE
Next stable release ?

### ISSUE THREAD

**Issue by seb2411** [jarves/jarves#12]
Hello !

I just discovered Jarves CMS. Really impressive.
What are the plan in terms of release and roadmap ?

Regards,
Sebastien.

**Comment by marcj** [comment]
Yeah, we're working on it :)

**Comment by muehre** [comment]
Would be interesting when you plan to release first stable version?! :)

**Comment by Ivan0905** [comment]
What is the status on this? Can i use it in production? I know you are saying it is not production ready, but are the current components somewhat stable?

**Comment by marcj** [comment]
yeah, it's stable but has no stable tag yet. I know some projects use it in production. Go use it, and provide feedback about its outcome.

**Comment by Adraesh** [comment]
It's really impressive ! Looking forward to use it on a real project.

**Comment by djkoza** [comment]
I'm realy waiting for it ! In my project - I must override a lot of Jarves methods because I want to call action method (custom code) method returns (for example) data what is needed for view. Think about this feature. Now programmer has problem when want handle action via custom code -> process data or something else.

**Comment by djkoza** [comment]
How can I handle Jarves Forms via custom code - what is the best way to do it? Documentation is so modest...
